rockspec/apisix-master-0.rockspec

```diff
@@ -54,6 +54,7 @@ dependencies = {
     "skywalking-nginx-lua-plugin = 1.0-0",
     "base64 = 1.5-2",
     "dkjson = 2.5-2",
+    "resty-redis-cluster = 1.02-4",
```

Comment:
the version is `1.02-4`? why so strange?

Reply:
it is the right version

Reply:
https://luarocks.org/modules/steve0511/resty-redis-cluster